src/ansible_compat/runtime.py

```diff
@@ -461,6 +466,22 @@ def _install_galaxy_role(
             link_path,
         )
 
+    def _update_env(self, varname: str, value: List[str], default: str = "") -> None:
+        """Update colon based environment variable if needed.
+
+        New values are added by inserting them to assure they take precedence
+        over the existing ones.
+        """
+        if value:
```

Comment:
```suggestion
        if not value: return
```
We can then dedent the remainder of the function. But I would vote for dropping the condition since none of the calculations are especially expensive and I would assume that in the vast majority of use cases, the `value` parameter will not be an empty list.